codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -109,6 +110,14 @@ protected Http2ConnectionHandler(Http2ConnectionDecoder decoder, Http2Connection
         decoder = new DefaultHttp2ConnectionDecoder(connection, encoder, frameReader);
     }
 
+    void sendPrefaceOnHandlerAdded(boolean sendPrefaceOnHandlerAdded) {
```

Comment:
This isn't really clear from a semantic perspective. Does it mean that the preface won't be sent or that it will be sent earlier or later? `httpClearTextUpgrade` is clearer from that perspective, as the preface will be sent from `onHttpServerUpgrade()`. That's very non-obvious given its current name.

Reply:
@ejona86 so you suggest changing the method name ? Any good idea ?

Reply:
`httpClearTextUpgrade` seemed strictly better, given the code is already aware of `onHttpServerUpgrade()` and that's what it's called on the builder.